docs/03-endpoints/api-admin/projects.md

```diff
@@ -862,6 +865,11 @@ Example response:
 ```
 
 Operates on the following properties:
- - `knora-admin:projectRestrictedViewSize`: the IIIF size value
- - `knora-admin:projectRestrictedViewWatermark`: the path to the watermark image. **Currently not used!**
 
+- `knora-admin:projectRestrictedViewSize`: the IIIF size value
+- `knora-admin:projectRestrictedViewWatermark`: the path to the watermark image. **Currently not used!**
+
+Caveat: The restricted view settings only take effect
+for user groups that have restricted view permissions (or no permissions at all)
+on the `knora-api:hasStillImageFileValue` value of StillImageFile resources.
+User groups with view permission or higher will see the image in full resolution.
```

Comment:
```suggestion
Note: Restricted view settings only take effect, if a user has "Restricted View" permission on an image.
```

In my opinion, this information is superfluous, because this is the exact definition of what "restricted view permission" means in our system. But if it gets added, I would keep it as concise as possible.

Reply:
Perhaps it might be obvious for an API-dev, but when I started working on the permissions of the "postcards" project, it wasn't obvious for me. I have accepted your suggestion. Could I get an approve now?

Reply:
Sure! thanks for the contribution.  
(for future reference it may help to re-request a review, then the PR shows up in the "requested reviews" section again, which is what I most regularly check)

Reply:
all good! :)  
my statement was just complicated for "you would have got the approve 10 minutes earlier with a re-request" ^^